blazor/rich-text-editor/import-export.md

```diff
@@ -83,13 +83,13 @@ You can import the RTF file into the editor by using file uploader component, an
 {% endhighlight %}
 {% endtabs %}
 
-![Import to RTF file](./images/blazor-import-rtf.png)
+![Blazor RichTextEditor import to RTF file](./images/blazor-richtexteditor-import-rtf.png)
 
 > [View Sample in GitHub](https://github.com/SyncfusionExamples/import-rtf-file-to-blazor-rich-text-editor).
 
-## Import text file to editor 
+## Import text file to editor
 
-The Rich Text Editor allows you to load an external text file in to the editor. You can read the text file from your path using [StremReader](https://docs.microsoft.com/en-us/dotnet/api/system.io.streamreader?view=net-6.0) class and assign it to the editor with the `@bind-Value` property.
+The Rich Text Editor allows you to load an external text file into the editor. You can read the text file from your path using the [StremReader](https://docs.microsoft.com/en-us/dotnet/api/system.io.streamreader?view=net-6.0) class and assign it to the editor with the `@bind-Value` property.
```

Comment:
The Rich Text Editor allows you to load an external text file into the editor. Read the text file from your path using the [StremReader](https://docs.microsoft.com/en-us/dotnet/api/system.io.streamreader?view=net-6.0) class and assign it to the editor with the `@bind-Value` property.